styleguide/global/voice.jade

```diff
@@ -0,0 +1,164 @@
+.container.standard-padding
+
+  .row.standard-padding-full
+    .col-md-12
+      h1.no-margin.left Voice and Tone
+      h4.red How do we sound
+
+  .row.half-padding-full
+    .col-md-12
+      h5.red.no-margin What does Voice and Tone
+      h2.no-margin Even Mean
+    .col-md-6
+      h5.red Voice
+      p is our style, our point of view, our personality.
+    .col-md-6
+      h5.red Tone
+      p is our attitude and how we are perceived by those who listen.
+    .col-md-12
+      p Our voice determines how we write and how we speak to our audience. Tone is how our voice is perceived by our audience. That’s what we mean by “tone of voice.”
+
+  .row
+    .col-md-6
+      h5.red Voice
+      p At its core, our voice is always human. Because that’s what we are, how we talk and who we talk to. And it doesn’t matter if it’s marketing copy, phone calls, emails or notes on the fridge (see how we said “fridge” and not “refrigerator?” Friendly, like that), Rackers talk to the world the way we want to be spoken to: with helpful words and phrases that are informative, simple, clear and easy to understand.
+      p But since we’re a technology company, sometimes there’s only one right word or phrase, and it has more than two syllables. We avoid needless jargon, but in the right context, we don’t flinch from using words like MapReduce or /.htaccess. And while our audience and customers include some very technical folks who want simple descriptions for how something works or affects them, they don’t want anything dumbed down.
+    .col-md-6
+      h5.red Tone
+      p Everything we say should create feelings of inspiration and empowerment in our audiences. We accomplish this by providing them with relevant insights and solutions, by using straightforward, relatable language, and by connecting with them on a personal level.
+
+  hr
+
+  .row.half-padding-full
+    .col-md-12
+      h5.red.no-margin Let's take a look at
+      h2.no-margin Some Examples
+
+  .row.half-padding-full
+    .col-md-6
+      h5.red.center LET’S TALK LIKE THIS
```

Comment:
Since the `<h5>` tag does all the shouting for us, can we convert this to proper case in the jade code itself?